showcase-app/test-cases.md

```diff
@@ -95,3 +95,30 @@ If you want to run the tests interactively, you use `npm run test:ui`.
   - Reload the page.
   - Verify that the active profile is still displayed.
 
+### 11. Login without extension using WebID
+
+- **Description**: Test the login flow using a WebID input field.
+- **Test Steps**:
+  - Register a profile.
+  - Load main page.
+  - Enter the profile's WebID in the input field.
```

Comment:
Who enters the WebID? The extension? The showcase app? The user?

Reply:
in this case (test) it's simulating the user entering the WebID manually

Reply:
every single step of every test case is an instruction for the test runner, so I don't see the confusion